### ISSUE TITLE
gall did not update correctly; got %ap-fill-under

### ISSUE THREAD

**Issue by ault011** [urbit/urbit#508]
Got this unfriendly looking print out during the latest merge. 

```
sync succeeded from %base on ~latlec-fonsel to %home
sync succeeded from %base on ~latlec-fonsel to %kids
sync succeeded from %kids on ~doznec to %base
[%vane %g /~latlec-fonsel/home/~2015.11.3..20.28.51..20af/arvo/gall ~filsen-tabtun]
  /~latlec-fonsel/home/~2015.11.3..20.28.51..20af/arvo/gall:<[2 1].[1,274 3]>
  /~latlec-fonsel/home/~2015.11.3..20.28.51..20af/arvo/gall:<[4 1].[1,274 3]>
  /~latlec-fonsel/home/~2015.11.3..20.28.51..20af/arvo/gall:<[5 1].[1,274 3]>
  /~latlec-fonsel/home/~2015.11.3..20.28.51..20af/arvo/gall:<[92 1].[1,274 3]>
  /~latlec-fonsel/home/~2015.11.3..20.28.51..20af/arvo/gall:<[93 1].[1,274 3]>
  /~latlec-fonsel/home/~2015.11.3..20.28.51..20af/arvo/gall:<[97 1].[1,274 3]>
  /~latlec-fonsel/home/~2015.11.3..20.28.51..20af/arvo/gall:<[98 1].[1,274 3]>
  /~latlec-fonsel/home/~2015.11.3..20.28.51..20af/arvo/gall:<[101 3].[1,178 5]>
  /~latlec-fonsel/home/~2015.11.3..20.28.51..20af/arvo/gall:<[102 3].[1,178 5]>
  /~latlec-fonsel/home/~2015.11.3..20.28.51..20af/arvo/gall:<[534 5].[1,177 7]>
  /~latlec-fonsel/home/~2015.11.3..20.28.51..20af/arvo/gall:<[535 5].[1,177 7]>
  /~latlec-fonsel/home/~2015.11.3..20.28.51..20af/arvo/gall:<[836 7].[847 48]>
  /~latlec-fonsel/home/~2015.11.3..20.28.51..20af/arvo/gall:<[837 7].[847 48]>
  /~latlec-fonsel/home/~2015.11.3..20.28.51..20af/arvo/gall:<[838 7].[847 48]>
  /~latlec-fonsel/home/~2015.11.3..20.28.51..20af/arvo/gall:<[839 7].[847 48]>
  /~latlec-fonsel/home/~2015.11.3..20.28.51..20af/arvo/gall:<[840 7].[847 48]>
  /~latlec-fonsel/home/~2015.11.3..20.28.51..20af/arvo/gall:<[841 7].[847 48]>
  /~latlec-fonsel/home/~2015.11.3..20.28.51..20af/arvo/gall:<[843 7].[847 48]>
  /~latlec-fonsel/home/~2015.11.3..20.28.51..20af/arvo/gall:<[843 21].[843 43]>
  /~latlec-fonsel/home/~2015.11.3..20.28.51..20af/arvo/gall:<[843 22].[843 36]>
  -find-limb.stop
  find-none
~mossug-tasled gets a merge
[%ap-fill-under [~latlec-fonsel %talk] ~latlec-fonsel 32]
[%ap-fill-under [~latlec-fonsel %talk] ~latlec-fonsel 32]
```

**Comment by philipcmonk** [comment]
```
~wictuc-folrex= all the symptoms seem consistent with a bad pill on gh from
~wictuc-folrex= eight days ago
--------[8.815]
~ramtev-wisbyt= there was another one?
~tasfyn-partyv wonders why it didn't autoreset
~wictuc-folrex= or possibly from five days ago
~tasfyn-partyv= do moons not autoreset?
~wictuc-folrex= autoreset only happens if there's a change to hoon.hoon
--------[8.820]
~ramtev-wisbyt= it loaded your gall update just fine
~wictuc-folrex= which there hasn't been since eight days ago
~wictuc-folrex= if any of the three pills since then are bad
~dilnem-nodfun= my moon's hoon.hoon does have ++stop in ++wa
~wictuc-folrex= that would explain it
--------[8.825]
~ramtev-wisbyt= i do have ++stop
~ramtev-wisbyt= and the !:
~tasfyn-partyv= ah, it's a sort of edge-triggered problem
~wictuc-folrex= or rather, an edge-triggered solution
~tasfyn-partyv= we have the new file but we didn't see a *change*
--------[8.830]
~ramtev-wisbyt= it fixed itself when i restarted that vere instance, fwiw
~ramtev-wisbyt= or at least hasn't triggered itself yet
~wictuc-folrex= it wouldn't be a problem until you changed a vane
~wictuc-folrex= try this:
~wictuc-folrex= (1) change gall trivially
--------[8.835]
~wictuc-folrex= predicition: fail to reload
~wictuc-folrex= (2) change hoon.hoon trivially
~wictuc-folrex= prediction: succeed to reset
~wictuc-folrex= (3) change gall trivially
~wictuc-folrex= prediction: succeed to reload
--------[8.840]
~ramtev-wisbyt= it's a pain that there isn't actually a "trivial change" command
~ramtev-wisbyt= we should have |touch
~latlec-fonsel= or |please
~latlec-fonsel= haha
~ramtev-wisbyt= predictions succeeded
--------[8.845]
~ramtev-wisbyt= perhaps actually recording a revision might be too much
~ramtev-wisbyt= maybe just trigger everything that *would* happen with a 
~ramtev-wisbyt= revision
~ramtev-wisbyt= just for debugging
```

tl;dr We pushed a bad pill recently, run `|reset` followed by `|reload %g`.
